ios/XMTPModule.swift

```diff
@@ -1460,6 +1480,12 @@ public class XMTPModule: Module {
 		  }
 		  return await client.contacts.isGroupDenied(groupId: groupId)
 		}
+
+		OnAppBecomesActive {
+			Task {
+				try await clientsManager.reconnectAllLocalDatabaseConnections()
+			}
+		}
```

Comment:
Is this a duplicate of line 1504

Reply:
sorry about that must have been the conflict when rebasing. Fixing.